projects/core/components/loader/loader-options.ts

```diff
@@ -0,0 +1,29 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+import {TuiSizeXS, TuiSizeXXL} from '@taiga-ui/core';
```

Comment:
Please use `@taiga-ui/core/types` instead `@taiga-ui/core`

Reply:
not sure if this is related

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/12021443/155679718-b250d2b3-2165-40ab-ba98-317cc2e22b32.png">

Reply:
This causes a circular dependency so that's probably the reason.

Reply:
fixed ✅ unit tests are being passed now. Thank you.